lms/djangoapps/grades/new/subsection_grade.py

```diff
@@ -80,7 +80,7 @@ def init_from_structure(self, student, course_structure, scores_client, submissi
                 student, descendant_key, course_structure, scores_client, submissions_scores, persisted_values={},
             )
         self.all_total, self.graded_total = graders.aggregate_scores(self.scores, self.display_name, self.location)
-        self._log_event(log.warning, u"init_from_structure", student)
+        self._log_event(log.info, u"init_from_structure", student)
```

Comment:
I agree - capturing more info per log entry and combining several of the CourseGrade entries will help a lot.